frontend/src/components/TableTaskRow.tsx

```diff
@@ -33,6 +41,63 @@ export const TableTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
     userInfoSelector,
     shallowEqual,
   );
+  const allUsers = useSelector<RootState, RoadmapUser[] | undefined>(
+    roadmapUsersSelector,
+    shallowEqual,
+  );
+  const allCustomers = useSelector<RootState, Customer[] | undefined>(
+    allCustomersSelector,
+    shallowEqual,
+  );
+  const [missingRatings, setMissingRatings] = useState<Customer[] | undefined>(
+    undefined,
+  );
+  const [missingDevRatings, setMissingDevRatings] = useState<
+    RoadmapUser[] | undefined
+  >([]);
+  const [userRatingMissing, setUserRatingMissing] = useState<Boolean>(true);
```

Comment:
```suggestion
  const [userRatingMissing, setUserRatingMissing] = useState<boolean>(true);
```
Yhtenäisempi muun koodin kanssa näin

Reply:
![image](https://user-images.githubusercontent.com/65648793/121904183-a5948e00-cd31-11eb-99c5-1a1557356a82.png)
Itellekkin tuli tää joskus yllätyksenä mutta Boolean, boolean, String, string yms. eivät ole sama asia vaan noi uppercaset on noita js luokkia (löytyy esim Number.isNan(..) yms)